index.html

```diff
@@ -1,66 +1,111 @@
 ---
+title_translation_key: homepage_page_title
```

Comment:
I think the frontpage should probably just use a default fallback for all the pages. That way you make sure to have the metadata of all pages make sense even if not explicitly set

Reply:
Yes

Reply:
Done!